From 8cd17baf849ccef00576c0a781bbd1c2c17f51ba Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Tue, 3 Jun 2014 15:33:37 -0700
Subject: [PATCH 1/2] kernel/power: match power hal function prototypes

Commit cacd529b63f3fe1070734fcc50b4f8491a051f3f
(suspend: Add power_hal suspend status macros) renamed
functions in the header files but did not rename the actual
functions. This renames the actual functions so they can
be used by others.

Change-Id: I82f33253156debc935b403a774713c0ef3a7862c
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 kernel/power/main.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/kernel/power/main.c b/kernel/power/main.c
index 80b38ef..574a1a8 100644
--- a/kernel/power/main.c
+++ b/kernel/power/main.c
@@ -632,7 +632,7 @@ static int __init pm_start_workqueue(void)
 static inline int pm_start_workqueue(void) { return 0; }
 #endif
 
-int register_power_HAL_suspend_device(struct device *dev)
+int register_power_hal_suspend_device(struct device *dev)
 {
 	if (!power_hal_kobj || !dev)
 		return -ENODEV;
@@ -640,13 +640,13 @@ int register_power_HAL_suspend_device(struct device *dev)
 	return sysfs_create_link(power_hal_kobj, &dev->kobj,
 			dev_name(dev));
 }
-EXPORT_SYMBOL(register_power_HAL_suspend_device);
+EXPORT_SYMBOL(register_power_hal_suspend_device);
 
-void unregister_power_HAL_suspend_device(struct device *dev)
+void unregister_power_hal_suspend_device(struct device *dev)
 {
 	sysfs_delete_link(power_hal_kobj, &dev->kobj, dev_name(dev));
 }
-EXPORT_SYMBOL(unregister_power_HAL_suspend_device);
+EXPORT_SYMBOL(unregister_power_hal_suspend_device);
 
 static int __init pm_init(void)
 {
-- 
2.0.0

